src/lib/filter.ts

```
@@ -1,4 +1,202 @@
+            raw: `"${expression.name}"`,
+          } satisfies jsep.Literal;
+        }
+
+        const resolvedValue = getFilterNodeValue(formattedExpression, context);
+        values.push(...Array.isArray(resolvedValue) ? resolvedValue : [ resolvedValue, ]);
+
+        return values;
+      }, []);
+
+    case 'Identifier':
+      const fieldName = filterAliases[node.name] || node.name;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)